src/uniprotkb/components/entry/tabs/genomic-coordinates/CoordinateTable.tsx

```diff
@@ -303,7 +304,6 @@ const CoordinateExtraContent = ({
             <th>Exon ID</th>
             <th>UniProtKB Isoform</th>
             <th>Position(s)</th>
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
I think we could just keep the exception at the exact line rather than for the whole file.